src/main/java/org/sonarsource/scanner/maven/bootstrap/MavenPlugin.java

```diff
@@ -37,6 +37,7 @@
 public class MavenPlugin {
 
   private Xpp3Dom configuration;
+  public int a;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make a a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](http://localhost:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarsource.scanner.maven%3Asonar-maven-plugin&pullRequest=1&issues=AWIp2JA7ACjrpjTBEmDR&open=AWIp2JA7ACjrpjTBEmDR)

[ISSUE_KEY]: AWIp2JA7ACjrpjTBEmDR